generators/setup-docker/templates/fabric-docker/commands-generated.sh

```diff
@@ -14,17 +14,17 @@ function installChaincodes() {
     %>
     printHeadline "Installing '<%= chaincode.name %>' on <%= chaincode.channel.name %>/<%= org.name %>/<%= peer.name %>" "U1F60E"
     <% if(!networkSettings.tls) { -%>
-    chaincodeInstall "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "<%= chaincode.channel.name %>" "<%= peer.fullAddress %>" "<%= rootOrg.ordererHead.fullAddress %>" "cli.<%= org.domain %>"
+    chaincodeInstall "cli.<%= org.domain %>" "<%= peer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>"
```

Comment:
since you already change it, maybe it would be good to pass all parameters in separate lines?